src/pages/api/makeRes.ts

```diff
@@ -1,41 +1,59 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
-import { Result } from "antd";
-import { responsiveArray } from "antd/lib/_util/responsiveObserve";
 import type { NextApiRequest, NextApiResponse } from "next";
 import { getDateInfo, makeRes } from "../../lib/backend/database-utils";
 
 type Data = {
-  name: string;
+	name: string;
 };
-
+//api handler
 const handler = async (req: NextApiRequest, res: NextApiResponse<Data>) => {
-  if (req.method == "POST") {
-    const firstName = req.body.firstName;
-    const lastName = req.body.lastName;
-    const email = req.body.email;
-    const language = req.body.language;
-    const course = req.body.course;
-    const middID = req.body.middID;
-    const resDate = req.body.resDate;
-    const type = req.body.type;
-    const is_cancelled = req.body.is_cancelled;
-    const on_waitlist = req.body.on_waitlist;
-    const attended = req.body.attended;
-    const data = await makeRes(
-      firstName,
-      lastName,
-      email,
-      language,
-      course,
-      middID,
-      resDate,
-      type,
-      is_cancelled,
-      on_waitlist,
-      attended
-    );
-  } else {
-  }
+	if (req.method == "POST") {
+		const firstName = req.body.firstName;
+		const lastName = req.body.lastName;
+		const email = req.body.email;
+		const language = req.body.language;
+		const course = req.body.course;
+		const middID = req.body.middID;
+		const resDate = req.body.resDate;
+		const type = req.body.type;
+		const is_cancelled = req.body.is_cancelled;
+		const on_waitlist = req.body.on_waitlist;
+		const attended = req.body.attended;
+        var waitlistBinary = 0;
```

Comment:
I refactored this endpoint into POST:/api/reservations for the same reasons as above.  You also need to make sure you validate your inputs.

The `var` keyword is essentially depreciated in modern JS.  You should use `let` or `const` instead unless you explicitly need variable hoisting.